### ISSUE TITLE
Error in price?

### ISSUE THREAD

**Issue by Maljor** [Csstenersen/node-red-contrib-nordpool-api#1]
When the price is above 1000 "NOK/MWh", the price is written as only 1 "NOK/MWh" in the array node. (Version 3.0.2)

**Comment by Csstenersen** [comment]
Problaby cause the number returns as a string with a space as this from nordpool. "1 000". I will look into this

**Comment by Csstenersen** [comment]
@Maljor Version 3.0.4 now published at node-red. please make it a try and give feedback before I close the issue.
Thanks for the feedback regarding this issue.

**Comment by Maljor** [comment]
Thanks for updating the version. At the moment the price is below 1000.

**Comment by Amli2011** [comment]
I ran into this problem, and it was not an update then. I did edit the js file myself, and added a relplace(" ", "") on line 143 and line 176.
My self edited file works, and I see the same edit in your file, but only on line 176. I have not tested your file.

Today the price in Molde was over 1000nok/MWh.


[nordpool-api.js.txt](https://github.com/Csstenersen/node-red-contrib-nordpool-api/files/5809923/nordpool-api.js.txt)
